airbyte-integrations/connectors/destination-elasticsearch-strict-encrypt/src/test-integration/java/io/airbyte/integrations/destination/elasticsearch/ElasticsearchStrictEncryptDestinationAcceptanceTest.java

```diff
@@ -103,6 +103,19 @@ protected JsonNode getConfig() {
         .build());
   }
 
+    protected JsonNode getUnsecureConfig() {
+
+      final JsonNode authConfig = Jsons.jsonNode(Map.of(
```

Comment:
This looks like a place where if it's being a copy of the `authConfig` from `getConfig` that it could be separated into a method like `getAuthConfig` for code reuse? 

```java
protected JsonNode getAuthConfig() {
  return Jsons.jsonNode(Map.of(
    "method", "basic",
    "username", "elastic",
    "password", "MagicWord"));
}
```

Reply:
Updated, thanks